code/datums/components/unusual_effect.dm

```diff
@@ -0,0 +1,53 @@
+
+/particles/unusual_effect
+	icon = 'icons/effects/particles/pollen.dmi'
+	icon_state = "pollen"
+	width = 100
+	height = 100
+	count = 1000
+	spawning = 4
+	lifespan = 0.7 SECONDS
+	fade = 1 SECONDS
+	grow = -0.01
+	velocity = list(0, 0)
+	position = generator(GEN_CIRCLE, 0, 16, NORMAL_RAND)
+	drift = generator(GEN_VECTOR, list(0, -0.2), list(0, 0.2))
+	gravity = list(0, 0.95)
+	scale = generator(GEN_VECTOR, list(0.3, 0.3), list(1,1), NORMAL_RAND)
+	rotation = 30
+	spin = generator(GEN_NUM, -20, 20)
+
+/datum/component/unusual_effect
+	dupe_mode = COMPONENT_DUPE_HIGHLANDER
+
+	COOLDOWN_DECLARE(glow_cooldown)
+
+/datum/component/unusual_effect/Initialize(color, include_particles = FALSE)
+	var/atom/movable/parent_movable = parent
+	if (!istype(parent_movable))
+		return COMPONENT_INCOMPATIBLE
+
+	parent_movable.add_filter("unusual_effect", 2, list("type" = "outline", "color" = color, "size" = 2))
+	if(include_particles)
+		parent_movable.particles = new /particles/unusual_effect()
```

Comment:
Maybe should use a `particle_holder`. Seems really likely this can override other stuff.

Reply:
fair enough, wasn't aware we had one